Mailer/app/api/v1/Handlers/Users/UnSubscribeHandler.php

```diff
@@ -45,13 +45,13 @@ public function handle($payload)
             return new JsonApiResponse(404, ['status' => 'error', 'message' => 'list not found']);
         }
 
-        $userSubscription = $this->userSubscriptionsRepository->getUserSubscription($list, $payload['user_id'], $payload['email']);
+        if (isset($payload['variant_id'])) {
+            $userSubscription = $this->userSubscriptionsRepository->getUserSubscription($list, $payload['user_id'], $payload['email']);
 
-        if ($userSubscription === false) {
-            return new JsonApiResponse(404, ['status' => 'error', 'message' => 'User subscription not found']);
-        }
+            if ($userSubscription === false) {
```

Comment:
Please use `if (!$userSubscription)`, so it doesn't break after update to Nette 3. (it changes `false` to `null` and we usually check `fetch()` results with `if (!$foo)` anyway.